helm/templates/frontend-nginx-configmap.yaml

```diff
@@ -33,9 +33,5 @@ data:
           index index.html;
           listen 8000;
           server_name _;
-
-          location = /favicon.ico {
-            return 204;
-          }
```

Comment:
Github is showing this change, but this file doen't exist in master any more. Perhaps GitHub is just a bit slow?

Reply:
Ah, it's a conflict that needs resolving I suspect :)

Reply:
yeah! I just removed them.